app/ide-desktop/lib/dashboard/src/components/MenuEntry.tsx

```diff
@@ -68,7 +70,7 @@ export default function MenuEntry(props: MenuEntryProps) {
     >
       <div className="flex items-center gap-menu-entry whitespace-nowrap">
         <SvgMask src={info.icon ?? BlankIcon} color={info.color} className="size-icon" />
-        {label ?? info.name}
+        {label ?? getText(`${action}Shortcut`)}
```

Comment:
This might be hard to find in refactoring later. Perhaps `action` should define a `shortcut` field with contents like `'FooActionShortcut'` and we reference that directly here like <code>getText(`${action.shortcut}`)</code>?

Similar to the TailwindCSS principle of not constructing class names by concatenating text, but rather by matching on possibilities. That is, `if item.size == small { "font-sm" } else { "font-lg" }` rather than `"font-" + item.size`.

Just a suggestion.